com.w1hkj.flcluster.yml

```
@@ -0,0 +1,44 @@
+app-id: com.w1hkj.flcluster
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: flcluster
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
```

```suggestion
  - --socket=x11
```

From what I know, FLTK does not support Wayland.